### PR TITLE
Update using dashboard guide - Closes 1528

### LINKED ISSUE #1528: Update `Using dashboard with the blockchain application`
### Description

* Update `Using dashboard with the blockchain application` to use v6 interfaces
* Update terminologies to Lisk SDK v6
* Update images using Lisk SDK v6

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1528

### CODE REVIEW

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 1]
```diff
@@ -1,203 +1,244 @@
...
 
 [source,bash]
 ----
-./bin/run start
+npm run build
 ----
 
-Wait until the application start has been completed.
-Now it is possible to access the dashboard on the browser under `localhost: 4005`
+.Start the blockchain client with custom_config.json
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc:93]: Why should the node be started with a custom config here?

Please explain to the reader, maybe by moving the IMPORTANT note from line 103 above this snippet.

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 2]
```diff
@@ -1,203 +1,244 @@
...
 ====
 To use this guide, it is assumed that the following criteria have been met:
 
-. Lisk Commander is installed, and a basic blockchain application is already initialized, as explained in the guide xref:{url_guides_setup}[].
+. Lisk Commander is installed, and a basic Lisk application is already initialized, as explained in the guide xref:{url_guides_setup}[].
 . A new module `HelloModule` is created, as explained in the guide xref:{url_guides_module}[].
-. A new asset `HelloAsset` is created, as explained in the guide xref:{url_guides_asset}[].
-. A new plugin `LatestHelloPlugin` is created, as explained in the guide xref:{url_guides_plugin}[].
 ====
+// . A new plugin `HelloInfoPlugin` is created, as explained in the guide xref:{url_guides_plugin}[].
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]: To make the guide usable for majority of users, please update the prerequisites and remove the point that the HelloModule is required, or remove prerequisites box completely and explain the remaining point in the intro text.

You can still use the Hello command below, by saying, "... by the example of the Hello client(link) ..." or something similar, to indicate this is not one of the default commands/ events.

Alternatively, you could think of adding one of the "On this page you will learn..." boxes on top.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 3]
```diff
@@ -1,203 +1,244 @@
...
 :url_guides_setup: build-blockchain/create-blockchain-client.adoc
-:url_intro_plugins: understand-blockchain/sdk/plugins.adoc
 
-// TODO: Update the page by uncommenting the hyperlinks once the updated pages are available.
+:url_references_dashboard_plugin: https://lisk.com/documentation/lisk-sdk/v6/references/typedoc/modules/_liskhq_lisk_framework_dashboard_plugin.html
+:url_gitHub_passphrase: https://github.com/LiskHQ/lisk-sdk-examples/blob/development/tutorials/hello/hello_client/config/default/passphrase.json
+:url_typedoc_crypto: https://lisk.com/documentation/lisk-sdk/v6/references/typedoc/functions/_liskhq_lisk_cryptography.address.getLisk32AddressFromAddress.html
+:url_dev_validator: https://github.com/LiskHQ/lisk-sdk-examples/blob/development/tutorials/hello/hello_client/config/default/dev-validators.json
+
+The {url_references_dashboard_plugin}[Dashboard plugin] offers various features that facilitate interaction with a Lisk application.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]: For link leading to external websites, please always let them open in a new tab:

```suggestion
The {url_references_dashboard_plugin}[Dashboard plugin^] offers various features that facilitate interaction with a Lisk application.
```

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 4]
```diff
@@ -1,203 +1,244 @@
...
-	},
-	{
-		"passphrase": "transfer alien ticket whisper face ability rally planet town brick profit solution",
-		"address": "80bd220ff01b6a248822b337a11be79da7fb43d2"
-	},
-	{
-		"passphrase": "broccoli coast fade over atom chimney skate symptom ten rug pave marble",
-		"address": "252eeaf82f6d713501e561ddd437bf00a903f415"
-	},
+You can use the {url_gitHub_passphrase}[passphrase] of the `genesis` account of `hello_client`.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
You can use the {url_gitHub_passphrase}[passphrase^] of the `genesis` account of `hello_client`.
```

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 5]
```diff
@@ -1,203 +1,244 @@
...
 
-In the response, it can be seen that the hello message of the account was updated successfully.
+To verify that the hello message was sent successfully, select the `hello_getHello` endpoint from the section *Call endpoint*.
 
-image::guides/dashboard/call_action_success2.png["Get account action success",400]
+The `hello_getHello` endpoint is part of the xref:{url_guides_module_endpoints}[HelloModule endpoints] and it returns the last sent hello message for the account address that is specified in the endpoint input.
 
-=== Invoking actions
+Provide the `address` to which the passphrase belongs.
+For `hello_client` the passphrase points to the first account in the {url_dev_validator}[dev-validators.json] file, which is located in the `config/default` directory of the hello client.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
For `hello_client` the passphrase points to the first account in the {url_dev_validator}[dev-validators.json^] file, which is located in the `config/default` directory of the hello client.
```

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 6]
```diff
@@ -1,203 +1,244 @@
...
+[TIP]
+==== 
+The value of the `senderAddress` retrieved in the `hello_newHello` event is in the `hex` format, which can be easily converted to the Lisk32 format through the `cryptography` package.
+For example:
+
+[source,js]
+----
+lisk.cryptography.address.getLisk32AddressFromAddress(Buffer.from("8ebe46d631ae4cc7ad14ba5235edde56c5f34ec6", 'hex'));
+----
+For more information, see {url_typedoc_crypto}[getLisk32AddressFromAddress].
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
For more information, see {url_typedoc_crypto}[getLisk32AddressFromAddress^].
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good already!

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 7]
```diff
@@ -1,203 +1,244 @@
...
 
-//=== Creating new accounts
 
-=== Subscribing to events
 
-At the bottom of the dashboard is the "Recent events" window, which allows the possibility to subscribe to various events by selecting them from a dropdown box.
+== Dashboard walkabout
 
-It is now possible to subscribe to the newly created event `hello:newHello`
+Go to http://localhost:4005 to access the Dashboard.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
Go to http://localhost:4005[^] to access the Dashboard.
```

**Review by Tschakki — APPROVED** [review]
One little remark, rest LGTM :+1:

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 8]
```diff
@@ -1,203 +1,247 @@
...
+
+Otherwise, you can install the Dashboard plugin like this:
 
 [source,bash]
 ----
 npm i @liskhq/lisk-framework-dashboard-plugin
 ----
 
-Now open `plugins.ts`, import the Dashboard plugin, and register it with the application as shown below:
+Now open the `plugins.ts` file, import the Dashboard plugin, and register it with the application as shown below:
```
Comment (mitsuaki-u) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc:62]:
```suggestion
Now open the `plugins.ts` file, import the Dashboard plugin and register it with the application as shown below:
```

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 9]
```diff
@@ -1,203 +1,247 @@
...
+It is essential to enable WS API and configure the `allowedMethods` property before registering the `DashboardPlugin`.
+For more information, see the configurations described in the xref:{url_guides_config_hello}[configuration guide].
+====
+
+After updating the *custom_config.json* file, build the blockchain client again:
 
 [source,bash]
 ----
-./bin/run start
+npm run build
```
Comment (mitsuaki-u) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc:90]:
```suggestion
yarn build
```

**Review by mitsuaki-u — APPROVED** [review]
minor comment approved

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc (hunk 9 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc:90]: Throughout the docs, we use the `npm run build`, if you agree we can keep it as is.

**Pull request merged**